app/routes/search.tsx

```diff
@@ -1,194 +1,52 @@
-                className="inline-block mb-1 text-xl font-bold leading-7 transition-colors duration-200 hover:text-deep-purple-accent-700"
-              >
-                Fitness Bootcamp: Get Fit and Stay Active
-              </a>
-              <p className="mb-3 text-xs font-bold tracking-wide">
-                <a
-                  href="/"
-                  className="transition-colors duration-200 text-blue-gray-900 hover:text-deep-purple-accent-700"
-                  aria-label="Category"
-                  title="traveling"
-                >
-                  Mon, May 22 - <br />
-                  6:00 pm PDT
-                </a>
-              </p>
-              <p className="mb-2 text-xs">
-                Break a sweat and achieve your fitness goals with our high-intensity workout session suitable for all
-                fitness levels.
-              </p>
-            </div>
-            <div className="h-1/2">
-              <img
-                src="https://res.cloudinary.com/djqdtvv7u/image/upload/v1684784670/fitness-meeting_b6vtmv.jpg"
-                className="object-cover w-full h-full pb-5 px-5"
-                alt=""
-              />
-            </div>
-          </div>
-          <div className="overflow-hidden rounded-2xl shadow-sm bg-primary flex flex-col">
-            <div className="p-5 border-t-0 text-white flex-grow">
-              <a
-                href="/"
-                aria-label="Category"
-                title="Fitness Bootcamp: Get Fit and Stay Active"
-                className="inline-block mb-1 text-xl font-bold leading-7 transition-colors duration-200 hover:text-deep-purple-accent-700"
-              >
-                Fitness Bootcamp: Get Fit and Stay Active
-              </a>
-              <p className="mb-3 text-xs font-bold tracking-wide">
-                <a
-                  href="/"
-                  className="transition-colors duration-200 text-blue-gray-900 hover:text-deep-purple-accent-700"
-                  aria-label="Category"
-                  title="traveling"
-                >
-                  Mon, May 22 - <br />
-                  6:00 pm PDT
-                </a>
-              </p>
-              <p className="mb-2 text-xs">
-                Break a sweat and achieve your fitness goals with our high-intensity workout session suitable for all
-                fitness levels.
-              </p>
-            </div>
-            <div className="h-1/2">
-              <img
-                src="https://res.cloudinary.com/djqdtvv7u/image/upload/v1684784670/fitness-meeting_b6vtmv.jpg"
-                className="object-cover w-full h-full pb-5 px-5"
-                alt=""
-              />
-            </div>
-          </div>
-        </div>
+    <div className="bg-secondary py-5 pb-20">
+      <div className="">
+        {events.length === 0 ? (
+          <p>No events found. Please, try another keywords. </p>
```

Comment:
In the future, it would be great to add some content instead of showing the semi-empty page.